### ISSUE TITLE
Implement image cropping in Online and Apps

### ISSUE THREAD

**Issue by nikslor** [CollaboraOnline/online#2092]
**Is your feature request related to a problem? Please describe.**
LibreOffice as well as other solutions such as GDocs offer the possibility to crop pictures.

In LibreOffice desktop this function can be found as shown in this screenshot:
![image](https://user-images.githubusercontent.com/1211300/115727463-e56b7480-a383-11eb-8ccc-46be43847126.png)

This is something many users expect from an office suite; they don't want to change the tool just to cut of a part of a picture.

**Describe the solution you'd like**
I propose to make this feature available in Online as well as the Apps.

Bonus points for an implementation where the shape of the cropping can be defined; see for example how Google does this. I often use this to cut of faces and put them in a circle shape:

![image](https://user-images.githubusercontent.com/1211300/115728828-22843680-a385-11eb-9674-ba43abeda24b.png)

**Comment by thebearon** [comment]
Seems reasonable.

**Comment by BrelLandJusticeFutures** [comment]
Agree. I'm implementing Collabora Online for our team; we're likely sticking with it, but this is a core feature we would frequently use in my team at least in Writer and Impress. Wherever possible, Collabora Online needs to mirror typical word processor functionality.

**Comment by Rash419** [comment]
This has been implemented and available in latest version.
